README.md

```diff
@@ -2,6 +2,17 @@
 
 ## Usage
 
+Ensure docker is running or if you are using Boot2Docker, run:
+
+```console
+boot2docker up
+boot2docker shellinit
```

Comment:
The eval version while setting it for the current shell session doesn't solve the problem though of it not being in the profile files and loading automatically in the future